src/components/Articles.jsx

```diff
@@ -0,0 +1,31 @@
+import getArticles from "../api";
+import { useEffect, useState } from "react";
+
+
+const Articles = () => {
+    const [articles, setArticles] = useState([])
+
```

Comment:
Could we add some loading functionality so the user gets some feedback before the items load?